src/internal/handle/styles.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../../internal/styles' as styles;
+@use '../../internal/styles/tokens' as awsui;
+
+.handle {
+  appearance: none;
+  background: transparent;
+  border: none;
+  padding: 0 awsui.$space-scaled-xxxs;
+  margin: 0 awsui.$space-scaled-xxs;
+
+  color: awsui.$color-text-interactive-default;
+
+  &:hover {
+    color: awsui.$color-text-interactive-hover;
+  }
+
+  &:focus-visible {
```

Comment:
@just-boris I know we recently made some changes to our focus handling. I wanted to double check that we are formally pivoting to using native CSS?

Reply:
This won't work in some browsers we support.

This is generally not allowed, but we can be flexible and make exceptions (it needs to be a team discussion first and I do not recall any threads)

Reply:
Thanks, addressed in [f981337](https://github.com/cloudscape-design/components/pull/804/commits/f981337b19e0e383f402590d59f380b2f1c053d5).

This was based on the internal handle component in board-components: https://github.com/cloudscape-design/board-components/blob/main/src/internal/handle/styles.scss#L37. I guess the justification in that case is that the focusVisible hook is not available there.